services/kommander/0.6.0/defaults/cm.yaml

```diff
@@ -74,14 +74,7 @@ data:
         prometheus-thanos-traefik: "0.0.1"
         thanos: "0.4.8"
     kommander-ui:
-      ingress:
-        enabled: true
-        extraAnnotations:
-          kubernetes.io/ingress.class: kommander-traefik
-          traefik.ingress.kubernetes.io/router.tls: "true"
-          traefik.ingress.kubernetes.io/router.middlewares: "${workspaceNamespace}-stripprefixes@kubernetescrd,${workspaceNamespace}-forwardauth@kubernetescrd"
-        path: /dkp/kommander/dashboard
-        graphqlPath: /dkp/kommander/dashboard/graphql
+      enabled: false
```

Comment:
maybe one more pr in kommander to remove kommander-ui as a subchart to go along with this and https://github.com/mesosphere/kommander-cli/pull/1035 ?

Reply:
https://github.com/mesosphere/kommander/pull/3315